pkg/analytics/executions.go

```diff
@@ -0,0 +1,95 @@
+package analytics
+
+import (
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+const ExecutionEventType = "bacalhau.execution_v1"
+
+type ExecutionEvent struct {
+	ExecutionID string `json:"execution_id,omitempty"`
+	JobID       string `json:"job_id,omitempty"`
+	EvalID      string `json:"eval_id,omitempty"`
+
+	// name of execution
+	Name string `json:"name,omitempty"`
+	// hash of the node name running execution
+	NodeNameHash string `json:"node_name_hash,omitempty"`
+	// hash of namespace execution is running in
+	NamespaceHash string `json:"namespace_hash,omitempty"`
+
+	// AllocatedResources
+	// NB: total execution resources for all tasks in execution
+	TotalCPUUnits    float64 `json:"total_cpu_units,omitempty"`
+	TotalMemoryBytes uint64  `json:"total_memory_bytes,omitempty"`
+	TotalDiskBytes   uint64  `json:"total_disk_bytes,omitempty"`
+	TotalGPUCount    uint64  `json:"total_gpu_count,omitempty"`
```

Comment:
lets include gpu types as well

Reply:
Okay, I can include this data in the event.

fwiw, I don't believe we currently set the `AllocatedResources` field of an execution, rather we read it from the `Job` field currently included in the execution.

Reply:
Upon second look, we actually never set the `GPUs` field of `Resources` when submitting a job. The `GPUs` field is included in the `Resources` for the purpose of sharing `NodeInfo` and `Labels`. 

Still, I'll include it anyways as you prefer.